lean4/src/putnam_1994_a3.lean

```diff
@@ -3,10 +3,9 @@ open BigOperators
 
 open Filter Topology
 
+-- Note: The formalization here differs slightly from the original problem statement, in that T is the entire triangle, not just the sides. We adopt the former interpretation because it is simpler to state and maintains the difficulty of the problem.
 theorem putnam_1994_a3
-(vec : ℝ → ℝ → (Fin 2 → ℝ))
-(T : Set (Fin 2 → ℝ) := convexHull ℝ {vec 0 0, vec 1 0, vec 0 1})
+(T : Set (EuclideanSpace ℝ (Fin 2)) := convexHull ℝ {![0,0], ![1,0], ![0,1]})
```

Comment:
Strictly you need to launder these `![_, _]` vectors through `WithLP.equiv`, to convert them to the right type. Probably the statement is still correct either way.

Reply:
Hm, will specify it anyways in the chance it would break in a future version of Lean. Thanks!